ts/components/session/PillDivider.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+interface ReceivedProps {
+  text: string;
+}
+type Props = ReceivedProps;
```

Comment:
any reason why you didn't do `interface Props` directly? 🤔

Reply:
Because of this tslint issue with SFC in react: https://github.com/microsoft/tslint-microsoft-contrib/issues/339